gpflow/config/__config__.py

```diff
@@ -143,10 +173,17 @@ def set_default_positive_minimum(value: float):
     set_config(replace(config(), positive_minimum=value))
 
 
+def set_default_summary_fmt(value: str):
+    formats = tabulate.tabulate_formats + ['notebook', None]
+    if value not in formats:
+        raise ValueError(f"Summary does not support '{value}' format")
+
+    set_config(replace(config(), summary_fmt=value))
+
+
 @contextlib.contextmanager
 def as_context(temporary_config: Optional[Config] = None):
-    '''Ensure that global configs defaults, with a context manager. Useful
-    for testing.'''
+    """Ensure that global configs defaults, with a context manager. Useful for testing."""
```

Comment:
This 
```suggestion
    """Ensure that global configs defaults, with a context manager. Useful for testing."""!
```
still doesn't make sense to me :( not your code but wanted to flag it up now that I noticed if you don't mind figuring out the intended meaning...

Reply:
Looks like I was drunk when were writing this :) @jahall skip it, I will fix it in a separate single PR with other doc fixes.